artemis/common_fixture.py

```diff
@@ -36,11 +36,11 @@ def clean_kirin_db():
         conn.commit()
 
         cur.execute(
-            "INSERT INTO contributor SELECT 'realtime.sherbrooke','ca-qc-sherbrooke','token_to_be_modified',"
+            "INSERT INTO contributor SELECT 'realtime.sherbrooke','ca-qc-sherbrooke','',"
```

Comment:
Do you want to keep this empty string?

```suggestion
            "INSERT INTO contributor SELECT 'realtime.sherbrooke','ca-qc-sherbrooke',"
```

Reply:
I actually realize that I didn't fully understand what my comment meant (it's an `INSERT`, not a list of random string). Anyway, thanks, the new code is nicer anyway :+1: